admin/export/class-export-keywords-query.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @package WPSEO\Admin\Export
+ */
+
+/**
+ * Class WPSEO_Export_Keywords_Query
+ *
+ * Creates a SQL query to gather all data for a keywords export.
+ */
+class WPSEO_Export_Keywords_Query {
+
+	/**
+	 * @var wpdb wpdb The wordpress database object.
+	 */
+	protected $wpdb;
+
+	/**
+	 * @var array[int]string The columns to query for, an array of strings.
+	 */
+	protected $columns;
+
+	/**
+	 * @var array[int]string The database columns to select in the query, an array of strings.
+	 */
+	protected $selects;
+
+	/**
+	 * @var array[int]string The database tables to join in the query, an array of strings.
+	 */
+	protected $joins = array();
+
+	/**
+	 * WPSEO_Export_Keywords_Query constructor.
+	 *
+	 * Supported values for columns are 'post_title', 'post_url', 'keywords', 'seo_score' and 'keywords_score'.
+	 * Requesting 'keywords_score' will always also return 'keywords'.
+	 *
+	 * @param array[int]string $columns The columns we want our query to return.
+	 * @param wpdb             $wpdb    A Wordpress Database object.
+	 */
+	public function __construct( $columns, $wpdb ) {
+		$this->columns = $columns;
+		$this->wpdb = $wpdb;
+	}
+
+	/**
+	 * Constructs the query and executes it, returning an array of objects containing the columns this object was constructed with.
+	 * Every object will always contain the ID column.
+	 *
+	 * @return array[int][string]string array of associative arrays containing the keys as requested in the constructor.
```

Comment:
See previous comments about return types. Also missing capitalization of the description.